packages/fxa-auth-client/lib/client.ts

```diff
@@ -507,7 +507,7 @@ export default class AuthClient {
     code: string,
     passwordForgotToken: hexstring,
     options: {
-      accountResetWithoutRecoveryKey?: boolean;
+      accountResetWithRecoveryKey?: boolean;
```

Comment:
This has been mismatched for who knows how long. In Backbone we're properly sending this up as `accountResetWithRecoveryKey`, but since there's no type safety we didn't catch that this was incorrect until now.

You can check the auth-server handler for this to verify it's expecting `request.payload.accountResetWithRecoveryKey`.